src/lib/datepicker/datepicker.md

```diff
@@ -287,6 +287,20 @@ export class MyApp {}
 
 <!-- example(datepicker-formats) -->
 
+#### Customizing the calendar header
+
+The header section of the calendar (the part containing the view switcher and previous and next
```

Comment:
Content projection wasn't feasible because it would have had to be passed through several layers `<mat-datepicker>` --> `<mat-datepicker-content>` --> `<mat-calendar>`. A template may have been workable, but there's a fair chunk of logic that goes with the header so a component seemed cleaner. At some point I want to break the datepicker into a bunch of composable pieces, we may be able to change how this works at that point